HW1/P3/P3.py

```diff
@@ -0,0 +1,10 @@
+from pyspark import SparkContext
+
+sc = SparkContext(appName="P3")
+words = sc.textFile('wlist.txt')
+counted_words = words.map(lambda x: (''.join(sorted(x)), [x])) \
+								.reduceByKey(lambda words1, words2: words1 + words2) \
+								.map(lambda (sorted_word, anagrams): (sorted_word, len(anagrams), anagrams)) \
+								.sortBy(lambda (sorted_word, count, anagrams): count, ascending=False)
+
```

Comment:
So many ways to skin a cat.  I also had three steps here, and none of them are the same as yours, but they do the same things.